themes/default/content/_index.md

```diff
@@ -40,95 +40,257 @@ overview:
       - /logos/tech/ci-cd/travis-ci.svg
 
 build:
-  title: Build
+  title: Write your configuration quickly
   description: |
-    Build cloud applications and infrastructure by combining the safety and reliability of [infrastructure as code](/what-is/what-is-infrastructure-as-code/) with the power of familiar programming languages and tools.
+    Pulumi helps you speed up your inner dev loop for IaC by allowing you to use an IDE giving you statement completion, real-time type checking, and interactive documentation working out the box. In addition you can leverage all the capabilities of the programming language of your choice to help you reduce the total number of lines of code you're writing.
   items:
     - title: Flexible and expressive
-      description: Loops, conditionals, functions, classes, and more.
-
-    - title: Be productive
-      description: Gets things done in seconds rather than hours.
+      description: |
```

Comment:
I feel like this bullet point section should just be removed to make it more streamlined. I feel like most of this section already made it into the paragraph above